docker-compose.dev.yml

```diff
@@ -47,4 +49,11 @@ services:
     env_file:
       - .env.docker.dev
     volumes:
-      - ./dev_db:/data/db
\ No newline at end of file
+      - ./dev_db:/data/db
+
+  prometheus:
```

Comment:
Unrelated here, but I notice that when I ran this file I couldn't access the localhost address because the Dockerfile doesn't specify `--host=0.0.0.0` for the dev container